main.py

```diff
@@ -5,20 +5,28 @@
 
 
 # See PyCharm help at https://www.jetbrains.com/help/pycharm/
+import csv
 import math
+import random
+
 import numpy as np
 
+import graph_io
+from trainingset import *
+from math import ceil
+
 from graph import Graph, Vertex, Edge
 from trainingset import TrainingSet
 
 # noinspection PyTypeChecker
-VertexList = list(Vertex)
+VertexList = [Vertex]
 
+TOP_EVENT_LABEL = "TE"
 
 def start():
     test_bowtie = create_test_bowtie()
-    training_set_event_tree = create_trainingset_event_tree(test_bowtie)
```

Comment:
Reverse this.  Wouter wanted this changed

Reply:
Did I? 🤔